catalog/USGS/USGS_NLCD_RELEASES_2021_REL_NLCD.jsonnet

```diff
@@ -0,0 +1,1019 @@
+    Database (NLCD) products: NLCD 1992, 2001, 2006, 2011, 2016, 2019, 
+    and 2021. Beginning with the 2016 release, land cover products were 
+    created for two-to-three-year intervals between 2001 and the most 
+    recent year. These products provide spatially explicit and reliable 
+    information on the Nation’s land cover and land cover change. 
+    NLCD continues to provide innovative, consistent, and robust 
+    methodologies for production of a multi-temporal land cover and 
+    land cover change database. The NLCD 2021 release is update based, 
+    so the Land Cover and Impervious Surface products released in 2019 
+    are unchanged and used directly with NLCD 2021 for change analysis 
+    though the NLCD timespan. Science products and the change index are 
+    updated and will need to be reacquired to contain the additional 2021 
+    change. These new products use a streamlined compositing process for 
+    assembling and preprocessing Landsat imagery and geospatial ancillary 
+    datasets; a temporally, spectrally, and spatially integrated land cover 
+    change analysis strategy; a theme-based post-classification protocol for 
+    generating land cover and change products; a continuous fields 
+    biophysical parameters modeling method; and a scripted operational system. 
+    The overall accuracy of the 2019 Level I land cover was 91%. Results from 
+    this study confirm the robustness of this comprehensive and highly automated 
+    procedure for NLCD 2021 operational mapping 
+    (see [doi:10.1080/15481603](https://doi.org/10.1080/15481603.2023.2181143) 
+    for the latest accuracy assessment publication). Questions about the NLCD 
+    2021 land cover product can be directed to the NLCD 2021 land cover 
+    mapping team at USGS EROS, Sioux Falls, SD (605) 594-6151 or 
+    mrlc@usgs.gov. See included spatial metadata for more details.
+    
+    Please see National Land Cover Database (NLCD) [2019 NLCD release](https://doi.org/10.5066/P9KZCM54)
+    for the 2019 release of NLCD which is used with the 2021 release for 
+    comparisons through the years. Also refer to the larger NLCD Community 
+    page for all things NLCD related National Land Cover Database (NLCD) 
+    [NLCD Community Page](https://www.sciencebase.gov/catalog/item/6345b637d34e342aee0863aa).
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.predecessor(predecessor_id, predecessor_url)
+  ],
+  keywords: [
+    'blm',
+    'landcover',
+    'mrlc',
+    'nlcd',
+    'usgs',
+  ],
+  providers: [
+    ee.producer_provider(
+      'United States Geological Survey and the 
+      Multi-Resolution Land Characteristics (MRLC) Consortium', 
+      'https://www.mrlc.gov'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-130.24, 21.75, -63.66, 50,
+                    '2021-01-01T00:00:00Z', '2022-01-01T00:00:00Z'),
+  summaries: {
+    'gee:schema': [

```

Comment:
do we have to include lines 83-114? or are we able to remove those? everything is listed in detail below already from line 116 on.

Reply:
Yes, we need these to generate the properties in the background.